.changeset/slow-vans-help.md

```diff
@@ -2,7 +2,7 @@
 "@fluid-tools/api-markdown-documenter": minor
 ---
 
-Enable `unicorn/prevent-abbreviations` eslint rule
+api-markdown-documenter: Enable `unicorn/prevent-abbreviations` eslint rule
```

Comment:
Good catch. The notes should really be focused on the API or behavior-relevant changes. I'll update the changeset
description. But regardless this was a
breaking change and should not have been made in main. @Josmithr

Reply:
Oh, actually, I'm wrong. This is an independent package, so the real bug here is that the changeset was loaded at all
since the package isn't in the client release group. I'll follow up on that.

Reply:
> breaking change and should not have been made in main

I didn't comment on that given the pre-1.0.0 status of the package and the fact it's outside the client release group. Which just now makes me realize... should this changeset exist here at all? The package is not part of the release 😅 .

Reply:
No, it shouldn't, and I noted that on the original change: https://github.com/microsoft/FluidFramework/pull/17368#discussion_r1330815397 I removed these changesets completely since they don't apply. When I ran `pnpm flub generate changelog -g client` it blew up as I expected, but `pnpm flub generate upcoming -g client -t minor --out RELEASE_NOTES/2.0.0-internal.6.4.0.md` processed them in error.

Reply:
Ah, I didn't realize it was problematic to add them. I'll keep that in mind in the future. That said, it would be nice to support our changeset tooling in individual packages too.